### DISCUSSION TITLE
Tekton support SVN Resource?

### DISCUSSION THREAD

**Discussion by ho-yasla** [tektoncd/catalog#1101] [Q&A]
Tekton support SVN Resource?

**Comment by vdemeester** [comment]
Hey @ho-yasla,

As PipelineResource are deprecated, the support for SVN (or git, or cvs, or any source code revision tools) is done with tasks (and use of workspace). For example, support for git is done through the [`git-clone`](https://github.com/tektoncd/catalog/tree/main/task/git-clone/0.9) task. For SVN, the idea would be very similar, a task that can checkout a SVN repository in a workspace to be used later on in a Pipeline.

Cheers
